Frontend/implementations/EpicGames/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "ps-application",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "build": "npx webpack",
+    "watch": "npx webpack --watch",
+    "serve": "webpack serve",
+    "install": "cd ../../library && npm run build-webpack && cd ../implementations/EpicGames && npm link ../../library",
+    "build-all": "npm run install && npm run build"
+  },
+  "devDependencies": {
+    "@epicgames/pixelstreamingfrontend": "^0.0.1",
+    "webpack-cli": "^5.0.1",
+    "webpack-dev-server": "^4.11.1",
+    "css-loader": "^6.7.3",
+    "html-loader": "^4.2.0",
+    "html-webpack-plugin": "^5.5.0",
+    "path": "^0.12.7",
+    "ts-loader": "^9.4.2",
```

```suggestion
    "ts-loader": "^9.4.2",
    "typescript": "^4.9.4",
```
Needs typescript as a devDependency. The build fails unless you have typescript installed globally.